content/storage-history/storage.textile

```diff
@@ -0,0 +1,109 @@
+---
+title:  Message Storage
+meta_description: "Explore the different ways Ably can handle Message Storage"
+
+---
+
+By default Ably will store messages for just two minutes so that they can be retrieved with the "History API":/storage-history/history in the event of a lost connection. However, there are a number of options available to increase this storage time. You can persist all of your messages (typically for 24 hours), persist the last message using the "channel rewind":/channels/options/rewind option, or use "integrations":/integrations to persist messages externally. 
+
+
+h2(#default-persistence). Default message storage - two minutes
+
+Accounts with the default message storage of two minutes can use the History API to recover messages in the original order they were sent, and applies to both "regular channel messages":/realtime/messages and "presence messages":/realtime/presence. For brief disconnections Ably automatically sends lost messages via the live subscription.
+
+The following diagram illustrates the default persistence of messages:
+
+<a href="/images/diagrams/history-default.png" target="_blank">
+  <img src="/images/diagrams/history-default.png" style="width: 100%" alt="Default Persistence">
+</a>
+
+Read more about the "History API":/storage-history/history or the "Realtime connection state recovery":/realtime/connection/#connection-state-recovery.
+
+
+h2(#all-message-persistence). Persist all messages - 24 or 72 hours
+
+If you need to retain messages for longer than the default two minutes you can enable "persisted history" when setting a "channel rule":/general/channel-rules-namespaces. When "persisted history is enabled":#persisted-history for a channel any messages will be stored on disk for 24 for free tiers or 72 hours for paid tiers.
```

Comment:
"...for 24 or 72 hours, depending on whether you are on a free or paid package."?